packages/components/src/components/radio-button/radio-button.stories.ts

```diff
@@ -8,11 +8,12 @@ const { overrideArgs } = storybookHelpers('sd-radio-button');
 const { generateTemplate } = storybookTemplate('sd-radio-button');
 
 /**
- * The `sd-radio-button` is a type of a radio-group-item similar to `sd-radio` which can be slotted insde of an `sd-radio-group`.  Radio button provides a button-style control for users to select one option from a group. Unlike `sd-radio`, `sd-radio-button` does not have an invalid state and should always have a pre-selected option (default value).
+ * It's a type of a radio-group-item similar to `sd-radio` which can be slotted inside of an `sd-radio-group`. Provides a button-style control for users to select one option from a group. Unlike `sd-radio`, `sd-radio-button` does not have an invalid state and should always have a pre-selected option (default value).
```

Comment:
Used to switch between different options or views by selecting one option from a group. 

Similar to `sd-radio`and can be slotted inside of an `sd-radio-group`. It does not have an invalid state and should always have a pre-selected option (default value).

Reply:
Add Related Components and link to sd-radio and sd-radio-group

Reply:
Updated